test/modules/toolbox/components/shareToolContent/ShareToolDialog.test.js

```diff
@@ -44,33 +50,40 @@ describe('ShareToolDialog', () => {
 		expect(element.shadowRoot.querySelectorAll('.share_item').length).toBe(1);
 	});
 
-	it('copies the url to the clipboard, when click', async (done) => {
-		const copySpy = spyOn(shareServiceMock, 'copyToClipboard').and.callFake(() => Promise.resolve());
+	it('copies the https url to the clipboard, when click', async (done) => {
+		const link = 'https://mock.url';
 
+		const copyToClipboardMock = spyOn(shareServiceMock, 'copyToClipboard').withArgs(link).and.returnValue(Promise.resolve());
 		const copyButton = element.shadowRoot.querySelector('.share_item .share_copy');
-
 		copyButton.click();
 
+		expect(copyButton).toBeTruthy();
+		expect(copyToClipboardMock).toHaveBeenCalledWith(link);
 		setTimeout(() => {
-			expect(copyButton).toBeTruthy();
-			expect(copySpy).toHaveBeenCalledWith('https://mock.url');
+			//check notification
+			expect(store.getState().notifications.latest.payload.content).toBe('map_contextMenuContent_clipboard_link_text map_contextMenuContent_clipboard_success');
+			expect(store.getState().notifications.latest.payload.level).toEqual(LevelTypes.INFO);
 			done();
 		});
 	});
 
-	it('logs a warning when copyToClipboard fails', async (done) => {
-		const copySpy = spyOn(shareServiceMock, 'copyToClipboard').and.callFake(() => Promise.reject());
-		const warnSpy = spyOn(console, 'warn');
-
-		const copyElement = element.shadowRoot.querySelector('.share_item .share_copy');
+	it('logs a warning when copyToClipboard fails using http ', async (done) => {
+		const link = 'http://mock.url';
+		element.shareUrl = link;
```

Comment:
property `shareUrl` is already set in line 41
Let's use its value here and in the previous test

Reply:
yes I see is already defined in line 41 with https, but in line 72 I just wanted to emphasize when using http instead of https there will be an error, for better understanding what s going on in this test